.gitignore

```diff
@@ -3,3 +3,5 @@ node_modules
 
 # WebStorm / PHPStorm
 .idea
+bin/act
+fortawesome-vue-fontawesome-*.tgz
```

Comment:
added so there are no accidents when testing locally.
This would be for the `fortawesome-vue-fontawesome-3.x.x.tgz` files